.infra/index.ts

```diff
@@ -427,13 +427,10 @@ if (isAdhocEnv) {
       env: [{ name: 'ENABLE_PRIVATE_ROUTES', value: 'true' }, ...commonEnv, ...jwtEnv],
       minReplicas: 2,
       maxReplicas: 2,
-      requests: {
+      limits: {
         memory: '256Mi',
         cpu: '25m',
       },
```

Comment:
Keep these split, but we can set mem request to `192Mi` and limit to `256Mi`.
CPU can also be adjusted down to `10m`